src/arch/x86_64/asm.rs

```diff
@@ -1,3 +1,6 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2020 Joe Richey
```

Comment:
I think it's probably more appropriate to assume Google copyright here as it was submitted under that email address.

Reply:
Thanks for your comment. Changed with commit 32c46d8.